### DISCUSSION TITLE
Feature Request - Exclude from instance block

### DISCUSSION THREAD

**Discussion by davidleebooth** [mastodon/mastodon#20189] [General]
Behavior: 
To be able to navigate to a users profile page, click exclude from instance block which then takes you to login to your account to set the exclusion, then if you need to block an instance but still follow the few you may do so. With many newcomers, friends may land on a hostile or blocked server at first, or for cases where there is that one good account in an instance of things you don't want to see.

**Comment by trwnh** [comment]
#14959
